jte-models/src/test/java/gg/jte/models/generator/TestModelExtension.java

```diff
@@ -14,75 +17,232 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.regex.Pattern;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import static gg.jte.extension.api.mocks.MockParamDescription.mockParamDescription;
-import static org.assertj.core.api.Assertions.assertThat;
 
 import static gg.jte.extension.api.mocks.MockConfig.mockConfig;
+import static gg.jte.extension.api.mocks.MockParamDescription.mockParamDescription;
 import static gg.jte.extension.api.mocks.MockTemplateDescription.mockTemplateDescription;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
 
+@DisplayName("jte-models Extension tests")
 public class TestModelExtension {
```

Comment:
I heavily refactored these tests, but I think they are easier to read now, and a few test cases were added. Again, some repetition.